pom.xml

```diff
@@ -333,6 +361,16 @@
 
   <!-- All of our needed dependencies -->
   <dependencies>
+    <dependency>
+      <groupId>io.confluent</groupId>
+      <artifactId>connect-utils</artifactId>
+      <version>0.6.4</version>
+    </dependency>
```

Comment:
we might need to add this specifically in snowflake-kafka-connector-private since its a private dep. 
in the final deployment, lets make sure we have this jar packaged.